gulp-core-build/src/tasks/BuildReceiptTask.ts

```diff
@@ -0,0 +1,134 @@
+import { GulpTask } from './GulpTask';
+import gulp = require('gulp');
+import * as fs from 'fs';
+import * as path from 'path';
+import * as child_process from 'child_process';
+
+export interface IBuildReceiptTask {
+}
+
+const PROCESS_OUTPUT_DELIMITER: string = '~X~X~X~X~X~X~';
```

Comment:
Might be good to use in the delimiter here, symbols which are disallowed in file paths